engine/src/main/java/org/hibernate/validator/cfg/defs/NormalizedDef.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Hibernate Validator, declare and validate application constraints
+ *
+ * License: Apache License, Version 2.0
+ * See the license.txt file in the root directory or <http://www.apache.org/licenses/LICENSE-2.0>.
+ */
+
+package org.hibernate.validator.cfg.defs;
+
+import org.hibernate.validator.cfg.ConstraintDef;
+import org.hibernate.validator.constraints.Normalized;
+
+import java.text.Normalizer;
+
+
+/**
+ * Constraint definition for {@link Normalized}.
+ * @author Craig Andrews
+ * @since 6.2.0
```

Comment:
This will have to be `6.1.6` as we will integrate the feature there. Same remark for all the occurrences.